awx/ui_next/src/screens/WorkflowApproval/shared/WorkflowApprovalStatus.jsx

```diff
@@ -28,11 +28,19 @@ function WorkflowApprovalStatus({ workflowApproval, i18n }) {
   if (workflowApproval.status === 'failed' && workflowApproval.failed) {
     return (
       <Tooltip
-        content={i18n._(
-          t`Denied by ${
-            workflowApproval.summary_fields.approved_or_denied_by.username
-          } - ${formatDateString(workflowApproval.finished)}`
-        )}
+        content={
+          workflowApproval.summary_fields?.approved_or_denied_by?.username
+            ? i18n._(
```

Comment:
Remove the `i18n._` here in favor of  ```t`string` ```

If you can get rid of the `i18n` object and `withI18n` in this file as well that would be good to do.